### ISSUE TITLE
make docs fails - doxygen complains about the help-list option

### ISSUE THREAD

**Issue by eyalroz** [doxygen/doxygen#7811]
I'm building doxygen 1.8.18 on Devuan GNU/Linux Beowulf. I managed to build most of it, and now I'm trying to `make docs` after enabling building documentation via CMake. I'm getting the following:
```
# cd /tmp/doxygen-1.8.18/build/examples && ../bin/doxygen class.cfg
: CommandLine Error: Option 'help-list' registered more than once!
LLVM ERROR: inconsistency in registered CommandLine options
```
My CMake options are:
```
 CMAKE_BUILD_TYPE                                                                                                                                                                                                                             
 CMAKE_INSTALL_PREFIX             /opt/versions/doxygen-1.8.18                                                                                                                                                                                
 Clang_DIR                        /usr/lib/cmake/clang-7                                                                                                                                                                                      
 DOC_INSTALL_DIR                  share/doc/packages/doxygen                                                                                                                                                                                  
 DOT                              /usr/bin/dot                                                                                                                                                                                                
 EPSTOPDF                         /usr/bin/epstopdf                                                                                                                                                                                           
 LANG_CODES                       AM;AR;BR;CA;CN;CZ;DE;DK;EN;EO;ES;FA;FI;FR;GR;HR;HU;ID;IT;JE;JP;KE;KR;LT;LV;MK;NL;NO;PL;PT;RO;RU;SC;SI;SK;SR;SV;TR;TW;UA;VI;ZA                                                                               
 LLVM_DIR                         /usr/lib/llvm-7/cmake                                                                                                                                                                                       
 MAKEINDEX                        /usr/bin/makeindex                                                                                                                                                                                          
 PDFLATEX                         /usr/bin/pdflatex                                                                                                                                                                                           
 build_app                        ON                                                                                                                                                                                                          
 build_doc                        ON                                                                                                                                                                                                          
 build_doc_chm                    OFF                                                                                                                                                                                                         
 build_parse                      ON                                                                                                                                                                                                          
 build_search                     OFF                                                                                                                                                                                                         
 build_wizard                     OFF                                                                                                                                                                                                         
 build_xmlparser                  OFF                                                                                                                                                                                                         
 english_only                     OFF                                                                                                                                                                                                         
 force_qt4                        OFF                                                                                                                                                                                                         
 use_libclang                     ON                                                                                                                                                                                                          
 use_sqlite3                      OFF                                                                                                                                                                                                         
 win_static                       OFF     
```

**Comment by 0x8000-0000** [comment]
Problem still exists in master.

**Comment by albert-github** [comment]
Looks like related to #6379

**Comment by albert-github** [comment]
Problem has nothing to do with creation of the documentation but just with the general running of doxygen.

This problem is also addressed in #6379, setting this issue to duplicate
